backend/app/Http/Controllers/Api/ProceduresApiController.php

```
@@ -0,0 +1,20 @@
+use App\Http\Controllers\Controller;
+use App\Models\Procedure;
+
+class ProceduresApiController extends Controller
+{
+
+    public function getProcedures()
+    {
+        return Procedure::paginate();
+    }
+
+    public function getProcedure($id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (Naming Rules)

(ShortVariable)